docs/testing-deployment-publishing/publishing/apple-appstore-deployment.md

```diff
@@ -10,56 +10,39 @@ keywords: [Apple App Store, Deployment, FlutterFlow, iOS]
+# Apple App Store Deployment
 
-:::info [Prerequisites]
+FlutterFlow allows you to deploy your apps directly to the App Store from within the platform. This guide covers all the necessary prerequisites, a step-by-step deployment process, and common troubleshooting tips.
 
+:::info[Prerequisites]
 - Create an [**Apple account**](https://appleid.apple.com/account?appId=632&returnUrl=https%3A//developer.apple.com/account/).
-- [**Purchase an Apple Developer membership**](https://developer.apple.com/programs/enroll/). Please see [**this link**](https://developer.apple.com/programs/) for more details on the Apple Developer program and how to sign up.
-- Set an App Launcher Icon for your app (under Settings & Integrations > General > App Assets). **The App Launcher icon can't be transparent or contain an alpha channel.**
-- We recommend testing your application on a real device. [**Here are instructions**](../running-your-app/local-run.md) on how to do this.
-
+- [**Purchase an Apple Developer membership**](https://developer.apple.com/programs/enroll/). Learn more about the program and enrollment process [here](https://developer.apple.com/programs/).
+- Set an App Launcher Icon for your app under **Settings & Integrations > General > App Assets**. **Note**: The launcher icon cannot be transparent or contain an alpha channel.
+- It's recommended to test your app on a real device before deployment. Follow [**these instructions**](../testing/local-run) to test your app locally.
 :::
 
 ## Deploy to App Store
 
-Deploying to App Store consists of the following steps:
-
-1. [Create a Bundle Identifier](#1-create-a-bundle-identifier)
-2. [Add a new app to App Store Connect](#2-add-a-new-app-to-app-store-connect)
-3. [Add your Apple App ID to FlutterFlow](#3-add-your-apple-app-id-to-flutterflow)
-4. [Generate your API key and add It to FlutterFlow](#4-generate-your-api-key-and-add-it-to-flutterflow)
-5. [Add your issuer ID to FlutterFlow](#5-add-your-issuer-id-to-flutterflow)
-6. [Add your Key ID to FlutterFlow](#6-add-your-key-id-to-flutterflow)
-7. [Deploy](#7-deploy)
-8. [Submit your app for App Store approval](#8-submit-your-app-for-app-store-approval)
+The App Store deployment involves the following steps:
 
 ### 1. Create a Bundle Identifier
 
-A Bundle Identifier (ID) is a unique **string** that identifies your app within the Apple ecosystem. It's typically formatted in reverse domain name notation, such as `com.example.myapp`.
+A **Bundle Identifier (ID)** is a **unique string** that identifies your app within the Apple ecosystem, typically formatted in reverse domain name notation like `com.example.myapp`.
 
-Please follow these steps to create a Bundle ID:
+To create a Bundle ID, visit the [**Certificates, IDs & Profiles**](https://idmsa.apple.com/IDMSWebAuth/signin?appIdKey=891bd3417a7776362562d2197f89480a8547b108fd934911bcbea0110d07f757&path=%2Faccount%2Fresources%2Fcertificates%2Flist&rv=1) page, add a new **App ID**, and provide these details:
```

Comment:
the URL has the app id key, is that expected?

Reply:
No. Thanks for catching this.